OWNERS_ALIASES

```diff
@@ -5,3 +5,4 @@ aliases:
     - gnufied
     - bertinatto
     - dobsonj
+    - mpatlasov
```

Comment:
@jsafrane @mpatlasov What about adding all team members while doing this chore? Same for other PRs that are currently opened.

Reply:
@RomanBednar , I really don't know which path is "correct": adding all team members vs. each team member adds himself/herself as needed. Let's wait for Jan's comment and we also may discuss it with the team online.

Reply:
@RomanBednar , we discussed this topic with Jan today. I was wrong assuming that each engineer has to add himself/herself to proper OWNERS files. You're right -- all team members must be in each OWNERS file. I'll update my PRs soon accordingly.

Reply:
@RomanBednar , we discussed this topic with Jan today. I was wrong assuming that each engineer has to add himself/herself to proper OWNERS files. You're right -- all team members must be in each OWNERS file. I'll update my PRs soon accordingly.

Reply:
Awesome, thank you @mpatlasov!